person.rb

```diff
@@ -0,0 +1,22 @@
+class Person
+  attr_accessor :id, :name, :age
+
+  def initialize(age, name = 'unknown', parent_permission: true)
```

Comment:
[OPTIONAL]
To allow the user to put `parent_permission` while creating a person instance different than `true` if they want to, Kindly make sure 
```suggestion
  def initialize(age, name = 'unknown', parent_permission= true)
```